src/query/v2/context.hpp

```diff
@@ -42,21 +43,27 @@ struct EvaluationContext {
   mutable std::unordered_map<std::string, int64_t> counters;
 };
 
-inline std::vector<storage::v3::PropertyId> NamesToProperties(const std::vector<std::string> &property_names,
-                                                              DbAccessor *dba) {
+inline std::vector<storage::v3::PropertyId> NamesToProperties(
+    const std::vector<std::string> &property_names, msgs::ShardRequestManagerInterface *shard_request_manager) {
   std::vector<storage::v3::PropertyId> properties;
   properties.reserve(property_names.size());
   for (const auto &name : property_names) {
-    properties.push_back(dba->NameToProperty(name));
+    if (shard_request_manager != nullptr) {
+      properties.push_back(shard_request_manager->NameToProperty(name));
+    }
   }
```

Comment:
```suggestion
  if (shard_request_manager != nullptr) {
    for (const auto &name : property_names) {
      properties.push_back(shard_request_manager->NameToProperty(name));
    }
  }
```

Reply:
What about these? Maybe leave the `TODO` in, but these "structural" changes are easy to do (change the order of `nullptr` check and the loop).

Reply:
Whoops I forgot about these